dataformat-aasx/src/main/java/org/eclipse/digitaltwin/aas4j/v3/dataformat/aasx/AASXDeserializer.java

```
@@ -128,7 +128,7 @@ public List<InMemoryFile> getRelatedFiles() throws InvalidFormatException, IOExc
             try {
                 files.add(readFile(aasxRoot, filePath));
             } catch (Exception e) {
-                logger.warn("Loading file {} failed and will not be included.", filePath, e);
+                logger.warn("Loading file {} failed and will not be included.", filePath);
```

What's the reasoning to hide error details from the user?